packages/server/src/games.ts

```diff
@@ -137,6 +137,9 @@ export async function updateGameState(io: IOServer, roomName: string, playerActi
   const [tempState, finalPlayerAction] = calculatePlayerAction(oldState, playerAction);
   const [finalState, isMatchFinished] = calculatePlayerTurn(tempState);
 
+  // TODO: When defined an explicit order in the model (and maybe state) this won't be needed
+  finalState.players.sort((playerA, playerB) => playerA.username.localeCompare(playerB.username));
```

Comment:
mmm isn't it very dangerous? Are we saying that the first two alphabetically ordered usernames are the active players?

Reply:
No, I am just ordering the players because one of the calculations changes the order. So, just to be sure I re-order here.

Reply:
I thought since we are doing player[0] player[1] in the FE it was relating to this bit but probably in the game state we only have active players right? all good then

Reply:
I think later one, we might broadcast the order explicitly. Like `order: ['user1`, 'user2', 'user3']`. When I came across this problem, I realised why boardgame.io has a state and a context. State is anything that has to do with the current state (duh) of the game that can change, context is the config/internal stuff of the game. Maybe something we can do later...